pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -187,27 +192,32 @@ private Representation getSegmentsForTable(
     Representation presentation;
     final JSONArray ret = new JSONArray();
 
+    String offlineTableName = TableNameBuilder.OFFLINE_TABLE_NAME_BUILDER.forTable(tableName);
```

Comment:
I think we should just ignore the active flag here, and get the segments from idealstate no matter what.
We need to add this under "REALTIME" and "OFFLINE" names just like we do in the table API.